samples/zigbee/light_switch/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8)
+
+add_definitions(-DZB_ED_ROLE)
```

Comment:
Why is it needed to add definition like this? Isn't this covered by Kconfig `CONFIG_ZIGBEE_ROLE_END_DEVICE` option?

Reply:
ping

Reply:
removed